view/ViewableData.php

```diff
@@ -384,17 +337,32 @@ protected function objCacheName($fieldName, $arguments) {
 	 * @return mixed
 	 */
 	protected function objCacheGet($key) {
-		if(isset($this->objCache[$key])) return $this->objCache[$key];
+		if(isset($this->objCache[$key])) {
+			return $this->objCache[$key];
+		}
+		return null;
 	}
 
 	/**
 	 * Store a value in the field cache
 	 *
 	 * @param string $key Cache key
 	 * @param mixed $value
+	 * @return $this
 	 */
 	protected function objCacheSet($key, $value) {
 		$this->objCache[$key] = $value;
+		return $this;
+	}
+
+	/**
+	 * Clear object cache
+	 *
+	 * @return $this
+	 */
+	protected function objCacheClear() {
```

Comment:
Are we able to invalidate caches key-by-key? I'd be a bit worried about the performance impacts of clean-sweep caching...